ui/src/main/js/page/stocks/index.js

```diff
@@ -21,11 +21,12 @@ class Stocks extends React.Component {
     this.onNavigate = this.onNavigate.bind(this);
     this.refreshCurrentPage = this.refreshCurrentPage.bind(this);
     this.onSearch = this.onSearch.bind(this);
+    this.updateStock = this.updateStock.bind(this);
   }
   componentDidMount() {
     this.loadFromServer(this.state.pageSize);
     stompClient.register([
-      {route: '/topic/updateStock', callback: this.refreshCurrentPage}
+      {route: '/topic/updateStock', callback: this.updateStock}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7290232)
```suggestion
      {route: "/topic/updateStock", callback: this.updateStock}
```